### DISCUSSION TITLE
what turbines or boilers or this it's monitor friendly? and how to connect dynamic tanks?

### DISCUSSION THREAD

**Discussion by PiasekTheProgramer** [MikaylaFischler/cc-mek-scada#525] [Support]
what turbines or boilers or this it's monitor friendly? and how to connect dynamic tanks?

**Comment by MikaylaFischler** [comment]
I have no idea what your first sentence is asking, but you connect dynamic tanks the same way as turbines, boilers, and everything that isn't a reactor. You configure using the `configure` command on the RTU after connecting them. Please read the links I gave you on your other post.

**Reply by MikaylaFischler** [reply]
You can also go watch the video on my channel about the RTU configurator.
